appeals/web/src/server/appeals/appeal-details/site-visit/site-visit.controller.js

```diff
@@ -38,6 +38,30 @@ const renderScheduleSiteVisit = async (request, response) => {
 			}
 		} = request;
 
+		// Nullish coalescing assignment https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment
+		visitType ??= appealDetails.siteVisit.visitType;
```

Comment:
Does this mean existing values from the API response will overwrite values in the POST request body? If so I think it should be the other way around

Reply:
Yes but only if they are not defined.

Reply:
Oh I see now, I was misunderstanding how nullish coalescing works